### DISCUSSION TITLE
How to create a arrow in Card

### DISCUSSION THREAD

**Discussion by wildsonc** [mantinedev/mantine#3863] [Q&A]
Is it possible to create an arrow on a Card? Same as Popover

![image](https://user-images.githubusercontent.com/72408418/226912758-d8024448-569e-4410-9ec8-e21af76e096b.png)
![image](https://user-images.githubusercontent.com/72408418/226912771-934e2ada-4024-4513-bedc-5f1af42b26e8.png)

**Comment by rtivital** [comment]
Yes, it is possible. You can add it by placing an absolutely positioned `::before` element that looks like an arrow on the side of the Card.
